src/routes/docs/advanced/platform/pro/+page.markdoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: article
+title: Pro
+description: Understand Appwrite's different pricing plans, behaviors, billing cycles, and limitations.
+---
+
+Appwrite Cloud's Pro plan is designed for professional developers or development teams that need to build applications at scale.
+When your application outgrows Appwrite's generous Starter plan you can switch to Pro plan to continue growing your app.
+You can learn more about Starter plan's generous resource limits on the [pricing page](https://appwrite.io/pricing).
+
+# Create a Pro plan organization {% #create-a-pro-plan-organizations %}
+Appwrite's plans are applied to an entire organization.
+You can head to the [pricing page](https://appwrite.io/pricing) and click **Start building** 
+or create a new organization from the Appwrite Console and select the **Pro plan**.
+
+## Trial {% #trial %} 
+Everh Appwrite Pro plan organization comes with a **14 day free trial**. 
+Your first billing period will start after your trial ends.
+
+## Switch to Starter plan {% #switch-to-pro-plan %}
+Head to the overview of your organization through the profile menu on the top right of your Appwrite Console.
+Under the **Billing** tab, you can click **Change plan** to change your plans.
+
+# Resource limits {% #resource-limits %}
+Each plan in Appwrite Cloud has a set of resource limits. You can find the details of these resource limits in the in the [pricing page](https://appwrite.io/pricing).
+
+You can purchase additional resources when you go over the resource limits to continue scaling your app.
```

Comment:
this seems odd, `you can purchase`, as until budget cap is reached, addons are auto applied ie. they don't need to purchase